packages/browser-wallet/src/popup/pages/TermsAndConditions/AcceptTerms.tsx

```diff
@@ -57,7 +59,7 @@ export default function AcceptTerms({ children, onSubmit }: Props) {
                                 description={
                                     <div>
                                         {t('form.termsAndConditionsDescription')}{' '}
-                                        <ExternalLink path={urls.termsAndConditions}>
+                                        <ExternalLink path={config?.url || urls.termsAndConditions}>
```

Comment:
I have the fallback in case the wallet-proxy doesn't respond, but maybe we should add proper handling.